app/components/App.js

```diff
@@ -75,7 +82,8 @@ export default class App extends Component {
      .then((results) =>{
       let returnMatches = this.helper.getPotentialMakes(results, this.state.completeVehicles)
       this.setState({
-        apiResults: returnMatches[1]
+        makeMatches: returnMatches[0],
+        apiResults: returnMatches[1],
       })
 
       let makes = returnMatches[0].map((makeData) =>{
```

Comment:
I was worried about that in the past... Let me take a look. I'm fairly certain the data isn't changing, so a const would work better.